themes/conventional-commits/README.md

```diff
@@ -0,0 +1,69 @@
+# hugo conventional commits theme
```

Comment:
is this a default README that was copied from another theme? would a contributor ever need to run any of these commands? I assume they'd instead be modifying the theme that we're already shipping with?